stories/topics/con-gcp-application-architecture.adoc

```diff
@@ -10,18 +10,9 @@ image::aap-on-gcp-architecture.png[GCP application architecture]
+
+The Foundation product includes two Execution Nodes running on the same VM instances as the {ControllerName} components (this is called a Hybrid Node configuration in {PlatformNameShort}).  Additional Execution Node offerings may be purchased to increase the scale (total number of managed nodes) the {PlatformNameShort} deployment is licensed to automate.  When deploying the Execution Node offerings into an existing {PlatformNameShort} Foundation deployment additional Execution Node VM instances will be deployed and automatically connected to the {ControllerName} of the Foundation deployment and can immediately begin processing automation tasks. The {PlatformNameShort} components are run as containers utilizing the Podman container runtime on the VM instances. The Podman runtime configuration is managed as a system service utilizing systemd to ensure uptime and availability, restarting any failed containers automatically. SELinux is enabled on the VM instances and is supported down to the container level. Additional operational automations are provided by the offering, available as a separate docker container for download from registry.redhat.io.  These additional operational automations include backup, restore, and upgrade.  Any Common Vulnerabilities and Exposures (CVEs) found in the RHEL OS base image, the {PlatformNameShort} containers, or any included packages will be addressed during upgrade of the {PlatformNameShort} offering by swapping out the base RHEL Google Cloud Machine Image with a newer version including all required software, packages, and containers, this is done automatically for you through the use of the included upgrade automation. Customers can take advantage of these operational automations to simplify the operational readiness of {PlatformNameShort} within their own corporate standards freeing themselves up to focus on developing Ansible Automation to manage their own infrastructure and applications rather than spending time developing automations to manage {PlatformNameShort}.
```

Comment:
Node offerings can be purchased...
systemd (wrap in backticks)
Common Vulnerabilities and Exposures (wrap in _)